state-reconstruct-fetcher/src/l1_fetcher.rs

```diff
@@ -256,7 +256,18 @@ impl L1Fetcher {
                             }
 
                             if let Some(tx_hash) = log.transaction_hash {
-                                hash_tx.send(tx_hash).await.unwrap();
+                                match hash_tx.send(tx_hash).await {
+                                    Err(e) => {
+                                        if cancellation_token.is_cancelled() {
+                                            tracing::debug!("Shutting down...");
```

Comment:
```suggestion
                                            tracing::debug!("Shutting down main handle...");
```

Reply:
for debugging, I'd prefer every message to be different... :-)

Reply:
That's fair, how about `Shutting down tx sender...` then?

Reply:
OK